src/test/scala/where/WhereSpec.scala

```diff
@@ -12,6 +12,48 @@ import org.scalatest._
 
 class WhereSpec extends WordSpec with Matchers {
 
+  "given a predicate" must {
+
+    val startNode = AnyNode()
+    val path = startNode --> AnyNode()
+    val referenceableMap: ReferenceableMap = Map(path -> "path")
+
+    "allow simple node conditions" in {
+      val predicate = All.nodes(path) { node =>
```

Comment:
you think it would be worth making this read more like `All nodesIn path where { node => ... }`?